### PR TITLE
[LD-832] Update PUT request URLs for contact, account

### PR DESCRIPTION
**Pull request by Kavya-24**
Screenshot 2023-10-23 at 4 07 50 PM
Screenshot 2023-10-23 at 4 08 02 PM
Jira: https://apollopde.atlassian.net/browse/LD-832
Slack: https://apolloio.slack.com/archives/C02Q41PP9HC/p1697579480485859
Make sure you've checked off all these things before submitting:

- [x] This pull request isn't for a company's fork, it's intended for the upstream Slate shared by everybody.
- [x] This pull request is submitted to the `master` branch.
- [x] If it makes frontend changes, this pull request has been tested in the latest version of Firefox, Chrome, IE, and Safari.

### CODE REVIEW

File: source/includes/_accounts.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ print(response.text)
 }
 ```
 
-`PUT https://api.apollo.io/v1/accounts`
+`PUT https://api.apollo.io/v1/accounts/YOUR_ACCOUNT_ID`
```
Comment (PavanSankavar) [inline source/includes/_accounts.md:153]: Do, We need to remove 'id' from parameters table below this?

File: source/includes/_contacts.md [hunk 2]
```diff
@@ -247,7 +247,7 @@ print(response.text)
 }
 ```
 
-`PUT https://api.apollo.io/v1/contacts`
+`PUT https://api.apollo.io/v1/contacts/YOUR_CONTACT_ID`
```
Comment (PavanSankavar) [inline source/includes/_contacts.md:250]: Do, We need to remove 'id' from parameters table below this?

**Review by PavanSankavar — COMMENTED** [review]
Added few review comments please check

File: source/includes/_accounts.md (hunk 1 above)
Comment (Kavya-24) [reply source/includes/_accounts.md:153]: Ya sure, removing that as well.

**Review by atul-tw — APPROVED** [review]
LGTM

File: source/includes/_accounts.md (hunk 1 above)
Comment (Kavya-24) [reply source/includes/_accounts.md:155]: Done, please check. @PavanSankavar

File: source/includes/_contacts.md (hunk 2 above)
Comment (Kavya-24) [reply source/includes/_contacts.md:252]: Done, please check. @PavanSankavar . Added a descriptor as well.

**Comment by Kavya-24** [comment]
Screenshot 2023-10-23 at 5 34 03 PM

**Pull request merged**